.github/workflows/slow.yaml

```diff
@@ -134,3 +134,58 @@ jobs:
         with:
           name: build-logs-macos
           path: btlayer-*.log
+
+  freebsd:
+    strategy:
+      matrix:
+        osversion:
+          - 14.1
+          - 13.3
+
+    runs-on: ubuntu-22.04
+    name: freebsd(${{ matrix.osversion }})
+
+    steps:
+      - name: Checkout Sources
+        uses: actions/checkout@v4
+
+      - name: Setup ccache
+        uses: hendrikmuhs/ccache-action@v1.2.14
+        with:
+          verbose: 2 # default 0
+          key: freebsd-${{ matrix.osversion }}
+
+      - name: Run test-builds
+        id: test-builds
+        uses: vmactions/freebsd-vm@v1
+        with:
+          usesh: true
+          release: ${{ matrix.osversion }}
+          prepare: |
+            pkg update
+            pkg install -y \
+              autoconf \
+              autoconf-archive \
+              automake \
+              bash \
+              ccache \
+              cppunit \
+              gmake \
+              libltdl \
+              libtool \
+              m4 \
+              nettle \
+              pkgconf
+
```

Comment:
From build-logs-freebsd-14.1/btlayer-02-maximus.log:

```
WARNING: Translation toolkit was not detected.
Translation of af disabled
```

FreeBSD build logs show that we are not testing translation-related code on FreeBSD. We are testing that code on some Linux distros (at least). If it does not require heroic efforts, we should test it on FreeBSD as well, but I do not insist on this fix.

P.S. It looks like Squid has a bug: `--enable-translations` does not result in a build failure when translations cannot be enabled. Fixing that bug is outside this PR scope, of course, but we cannot rely on that ./configure option alone to make sure that translations are enabled (until that bug is fixed).